test/general/lambda_naming.pass.cpp

```diff
@@ -33,7 +33,7 @@ int main() {
     auto buf_begin = oneapi::dpl::begin(buf);
     auto buf_end = buf_begin + n;
 
-    const auto policy = TestUtils::default_dpcpp_policy;
+    const auto policy = TestUtils::get_dpcpp_test_policy();
```

Comment:
In fact there is not any parameter to pass into `get_dpcpp_test_policy()` calls...
So, we most likely change only `TestUtils::default_dpcpp_policy` definition, instead of changing `TestUtils::default_dpcpp_policy` by `TestUtils::get_dpcpp_test_policy()` in several dozen files....

Reply:
Previously, in `TestUtils::default_dpcpp_policy` we doesn't used the instance of `sycl::queue` created in `get_test_queue()`

Reply:
We can just change `TestUtils::default_dpcpp_policy` definition  - to add usage of the instance of `sycl::queue`, like created in `get_test_queue()`.

Reply:
good catch, fixed

Reply:
> We can just change TestUtils::default_dpcpp_policy definition - to add usage of the instance of sycl::queue, like created in get_test_queue().

It can make a false impression that `dpcpp_default` is being tested. I do not strongly insist, but I prefer having non-ambiguis names at the expense of more changes in the PR.